internal/receiver/smartagentreceiver/README.md

```diff
@@ -40,35 +43,43 @@ receivers:
     type: postgresql
     host: mypostgresinstance
     port: 5432
-    dimensionClients: [signalfx]
+    dimensionClients:
+      - signalfx  # references the SignalFx Exporter configured below
   smartagent/processlist:
     type: processlist
   smartagent/kafka:
     type: collectd/kafka
     host: mykafkabroker
     port: 7099
     clusterName: mykafkacluster
-    intervalSeconds: 1
+    intervalSeconds: 5
 
 processors:
   resourcedetection:
-    detectors: [system]
+    detectors:
+      - system
 
 exporters:
   signalfx:
 
 service:
   pipelines:
     metrics:
-      receivers: [smartagent/postgresql, smartagent/kafka]
-      processors: [resourcedetection]
-      exporters: [signalfx]
+      receivers:
+        - smartagent/postgresql
+        - smartagent/kafka
+      processors:
+        - resourcedetection
+      exporters:
+        - signalfx
     logs:
-      receivers: [smartagent/processlist]
-      processors: [resourcedetection]
-      exporters: [signalfx]
+      receivers:
+        - smartagent/processlist
+      processors:
+        - resourcedetection
+      exporters:
+        - signalfx
 ```
 
-The full list of settings exposed for this receiver are documented for
-[each monitor](https://github.com/signalfx/signalfx-agent/tree/master/docs/monitors), and the implementation is
-[here](./config.go)
+For a more detailed description of migrating your Smart Agent monitor usage to the Splunk distribution of
+OpenTelemetry Collector please see the [migration guide](../../../docs/signalfx-smart-agent-migration.md).
```

Comment:
"For a more detailed description of migrating your Smart Agent monitor usage to the Splunk Distribution of
OpenTelemetry Collector, see the [migration guide](../../../docs/signalfx-smart-agent-migration.md)."